SocialFilter/sections/specific.txt

```diff
@@ -11404,6 +11404,7 @@ sportsseoul.com,sportsseoul.co.kr##.left_socialWap
 sportsseoul.com,sportsseoul.co.kr##.right_iconMenu
 viva100.com##.left_top_sns
 ygosu.com###sns_bar
+hardcoregamer.com##.btn-flipboard
```

Comment:
@northis please take a look at my comments https://github.com/AdguardTeam/AdguardFilters/pull/176711#discussion_r1559232943.
Btw, those elements are part of the share, so we can block it.